src/include/distributed/citus_nodes.h

```diff
@@ -17,9 +17,10 @@
  *
```

Comment:
Can we add a two sentence overview about what has to be done to add a new node type here?